### PR TITLE
Add PyCon India 2016 - Workshop Introduction blog

### PR DESCRIPTION
**Pull request by meganeshkadam**

### CODE REVIEW

File: content/blog/pycon-india-2016-workshop-introduction.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Title: PyCon India 2016 - Workshop Introduction
+Date: 2016-08-17 15:00
+Category: 2016
+Slug: workshop-introduction-2016
+Summary: PyCon India 2016 - Workshop Introduction
+
+
+The workshops at [PyCon India](https://in.pycon.org/2016/) will be held on 23rd September, 2016 and are two and half hour long of hands-on sessions, so be sure not to miss it.
+
```
Comment (harisibrahimkv) [inline content/blog/pycon-india-2016-workshop-introduction.md]: The workshops at [PyCon India](https://in.pycon.org/2016/) will be held on 23rd September, 2016. They are hands-on sessions of 2.30hr duration each and are taught by experts in the field. You don't want to miss them.

File: content/blog/pycon-india-2016-workshop-introduction.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+There will be three workshops running parallely in different Rooms, make sure that your interested Workshop(s) do not conflict.
+
```
Comment (harisibrahimkv) [inline content/blog/pycon-india-2016-workshop-introduction.md]: We have selected a total of 6 workshops into this year's conference. There will be 3 workshops in parallel during the morning session and 3 of them in parallel during the afternoon session. Please make sure that you avoid schedule conflicts while purchasing tickets for the workshops.

File: content/blog/pycon-india-2016-workshop-introduction.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+There are few choices when it comes to Workshop:
+
```
Comment (harisibrahimkv) [inline content/blog/pycon-india-2016-workshop-introduction.md]: Following are the accepted workshops:

File: content/blog/pycon-india-2016-workshop-introduction.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- [Productive Coding with PyCharm ](https://in.pycon.org/cfp/2016/proposals/productive-coding-with-pycharm~aA11a/) by pauleveritt
+
+- [Demystifying the Django REST Framework](https://in.pycon.org/cfp/2016/proposals/demystifying-the-django-rest-framework~dBNxd/) by Haris Ibrahim K. V.
+
+- [Scaling Django with Kubernetes](https://in.pycon.org/cfp/2016/proposals/scaling-django-with-kubernetes~boONb/) by Saket
+
+
+More details about Workshops and Selected talks available at [PyCon India 2016 - Proposals](https://in.pycon.org/cfp/2016/proposals/#proposals)
+
```
Comment (harisibrahimkv) [inline content/blog/pycon-india-2016-workshop-introduction.md]: Further details about workshops and selected talks are available at [PyCon India 2016 - Proposals](https://in.pycon.org/cfp/2016/proposals/#proposals)

File: content/blog/pycon-india-2016-workshop-introduction.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+Tickets to the Workshops are available [here](https://in.explara.com/e/pycon-india-2016)
+
```
Comment (harisibrahimkv) [inline content/blog/pycon-india-2016-workshop-introduction.md]: Are the tickets already available? I can't see them on the Explara page. Also, you might want to mention the capacity of each workshop and ask people to grab their tickets as soon as possible if they don't want to miss out!

File: content/blog/pycon-india-2016-workshop-introduction.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+Title: PyCon India 2016 - Workshop Introduction
+Date: 2016-08-17 15:00
+Category: 2016
+Slug: workshop-introduction-2016
+Summary: PyCon India 2016 - Workshop Introduction
+
+
+The workshops at [PyCon India](https://in.pycon.org/2016/) will be held on 23rd September, 2016. They are hands-on sessions of 2.30hr duration each and are taught by experts in the field. You don't want to miss them!
```
Comment (vnbang2003) [inline content/blog/pycon-india-2016-workshop-introduction.md]: Workshop are 3 hours session.

Comment (vnbang2003) [inline content/blog/pycon-india-2016-workshop-introduction.md]: please check spell mistakes

File: content/blog/pycon-india-2016-workshop-introduction.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+We have selected a total of 6 workshops into this year's conference. There will be 3 workshops in parallel during the morning session and 3 of them in parallel during the afternoon session. Please make sure that you avoid schedule conflicts while purchasing tickets for the workshops.
```
Comment (vnbang2003) [inline content/blog/pycon-india-2016-workshop-introduction.md]: @harisibrahimkv @kracekumar Looks like we need you helping hands .Can you guys provide feedback ?

Comment (harisibrahimkv) [inline content/blog/pycon-india-2016-workshop-introduction.md]: @vnbang2003 I did provide feedback on the initial draft. Except for changing the 2.30hrs to 3hrs, everything else looks fine.

Comment (vnbang2003) [inline content/blog/pycon-india-2016-workshop-introduction.md]: @harisibrahimkv Thanks that change Gora will be providing feedback.

**Pull request closed without merging**